fees/veda.ts

```diff
@@ -0,0 +1,285 @@
+import { Interface } from "ethers"
+import { Adapter, FetchOptions, FetchResultV2 } from "../adapters/types"
+import { CHAIN } from "../helpers/chains"
+import * as sdk from '@defillama/sdk'
+
+/**
+ *
+ * Veda reinvest staked assets into others platforms and earning yield
+ * Yields are distributed to stakers and Veda protocol:.
+ * 
+ * Veda takes these fees:
+ * - Performance fees, a percentage amount of yield (config per vault)
+ * - Platform fees, a basic percentage amount of total assets in the vault (config per vault)
+ */
+
+const methodology = {
+  Fees: 'Total yields are genearted by staking assets.',
+  SupplySideRevenue: 'The amount of yields are distibuted to stakers.',
+  ProtocolRevenue: 'The amount of yields are distibuted to Veda Protocol.',
+}
+
+const BoringVaultAbis = {
+  //vault
+  hook: 'function hook() view returns(address)',
```

Comment:
our sdk supports a shorthand (non-standard) abi format for functions that have one output and no input,

 `function hook()...` can be written as `address:hook`
 `function decimals()` -> `uint8:decimals`